lib/core/public/load.js

```diff
@@ -1,6 +1,9 @@
-/*global Audit, runRules, cleanupPlugins */
+/*global Audit */
```

Comment:
Because `Audit` is not modularised yet - https://github.com/dequelabs/axe-core/pull/2158. I will add a todo note, so we do not forget to do such changes.